### PR TITLE
Add amounts to Legend

### PR DESCRIPTION
**Pull request by sulfurious**
Add amounts to Legend

### CODE REVIEW

**Comment by grgur** [comment]
You're the legend!

File: client/components/DonutChart/Legend.js [hunk 1]
```diff
@@ -1,17 +1,24 @@
...
+import formatAmount from 'utils/formatAmount';
 import styles from './styles.scss';
 
-const Legend = ({ data, color, dataLabel, dataKey }) => (
+const Legend = ({ data, color, dataValue, dataLabel, dataKey }) => (
   <ul className={styles.legend}>
     {data.map(
-      (item, idx) => <li style={{ color: color(idx) }} key={item[dataKey]}>{item[dataLabel]}</li>
+      (item, idx) => (
+        <li style={{ color: color(idx) }} key={item[dataKey]}>
```
Comment (grgur) [inline client/components/DonutChart/Legend.js]: This should be a functional component for readability purposes

File: client/components/DonutChart/styles.scss [hunk 2]
```diff
@@ -14,4 +16,10 @@
 .legend {
   font-size: larger;
   font-weight: bold;
-}
\ No newline at end of file
+}
+
+.value {
+  font-size: smaller;
```
Comment (grgur) [inline client/components/DonutChart/styles.scss]: can we use a `em` or `rem` value here?

File: client/components/DonutChart/styles.scss [hunk 3]
```diff
@@ -14,4 +16,10 @@
 .legend {
   font-size: larger;
```
Comment (grgur) [inline client/components/DonutChart/styles.scss]: em/rem here too

File: client/components/DonutChart/index.js [hunk 4]
```diff
@@ -87,8 +89,9 @@ class DonutChart extends Component {
         <Legend
```
Comment (grgur) [inline client/components/DonutChart/index.js]:
```jsx
<Legend {...{data, dataValue, dataLabel, dataKey}} color={colorFn} />
```

**Comment by sulfurious** [comment]
@grgur PR changes are in

File: client/components/DonutChart/LegendItem.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  </li>
+);
+
+LegendItem.propTypes = {
+  color: PropTypes.string.isRequired,
+  value: PropTypes.string.isRequired,
+  label: PropTypes.string.isRequired,
+};
+
+export default LegendItem;
```
Comment (grgur) [inline client/components/DonutChart/LegendItem.js]: new line please :)

**Comment by grgur** [comment]
muy bien 👍

File: client/components/DonutChart/LegendItem.js (hunk 5 above)
Comment (sulfurious) [inline client/components/DonutChart/LegendItem.js]: You're killing me ;-)

**Pull request merged**